tools/eslint-rules/inspector-check.js

```diff
@@ -11,16 +11,21 @@ const utils = require('./rules-utils.js');
 // Rule Definition
 //------------------------------------------------------------------------------
 const msg = 'Please add a skipIfInspectorDisabled() call to allow this ' +
-            'test to be skippped when Node is built \'--without-inspector\'.';
+  'test to be skippped when Node is built \'--without-inspector\'.';
 
 module.exports = function(context) {
   const missingCheckNodes = [];
+  const commonModuleNodes = [];
   var hasInspectorCheck = false;
 
   function testInspectorUsage(context, node) {
     if (utils.isRequired(node, ['inspector'])) {
       missingCheckNodes.push(node);
     }
+
+    if (utils.isCommonModule(node)) {
+      commonModuleNodes.push(node);
```

Comment:
Since there is only one required (and only one should be present) using an array here is actually not necessary and a simple assign would be enough as well.

Reply:
Yup correct. Fixing this.